scripts/get-early-access.js

```
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+
+/**
+ * Simple script to retrieve early access emails from Redis
+ * Usage: node scripts/get-early-access.js
+ *
+ * Requires UPSTASH_REDIS_REST_URL and UPSTASH_REDIS_REST_TOKEN to be set
+ */
+
+const { Redis } = require('@upstash/redis');
```

ESLint

[error] 'require' is not defined.

(no-undef)